<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupbox_adc0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC0</string>
    </property>
    <widget class="QLCDNumber" name="lcd_adc0">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>232</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget0" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkbox_adc0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_adc1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_adc2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_adc3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_adc7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_adc4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_adc5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_adc6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupbox_adc1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC1</string>
    </property>
    <widget class="QLCDNumber" name="lcd_adc1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>232</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget1" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupbox_adc3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC3</string>
    </property>
    <widget class="QLCDNumber" name="lcd_adc3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>232</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupbox_adc2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC2</string>
    </property>
    <widget class="QLCDNumber" name="lcd_adc2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>232</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupbox_adc5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC5</string>
    </property>
    <widget class="QLCDNumber" name="lcd_adc5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>232</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget5" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupbox_adc6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC6</string>
    </property>
    <widget class="QLCDNumber" name="lcd_adc6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>232</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget6" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupbox_adc6_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>310</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC7</string>
    </property>
    <widget class="QLCDNumber" name="lcd_adc7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>232</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget7" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupbox_adc4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC4</string>
    </property>
    <widget class="QLCDNumber" name="lcd_adc4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>232</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget4" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
